content/api/rest-api.textile

```diff
@@ -675,6 +675,32 @@ bc[json]. [{
   extras: <optional extras object>,
 }]
 
+h3(#annotations-client-summary). Retrieve annotation summary for a specific clientId
+
+h6. GET rest.ably.io/channels/@<channelId>@/messages/@<messageSerial>@/client-summary
+
+Retrieve the annotation summary for a specific clientId for a specific message identified by its serial.
+
+Example request:
+
+bc[sh]. curl https://rest.ably.io/channels/rest-example/messages/01726585978590-001@abcdefghij:001/client-summary?forClientId=client1 \
+ -u "{{API_KEY}}"
+
+h5. Parameters
+
+- forClientId := _client1_ The clientId to retrieve the annotation summary for. It defaults to the clientId of the authenticated client.
```

Comment:
can I ask why we ended up with `forClientId` on this endpoint? Every other rest endpoint that accepts a client id parameter uses `clientId`